data/Ames_Mutagenicity/meta.yaml

```diff
@@ -0,0 +1,43 @@
+name: Ames_Mutagenicity
+description: Mutagenicity means the ability of a drug to induce genetic alterations.
+  Drugs that can cause damage to the DNA can result in cell death or other severe
+  adverse effects. Nowadays, the most widely used assay for testing the mutagenicity
+  of compounds is the Ames experiment which was invented by a professor named Ames.
+  The Ames test is a short-term bacterial reverse mutation assay detecting a large
+  number of compounds which can induce genetic damage and frameshift mutations. The
+  dataset is aggregated from four papers.
+targets:
+- id: mutagenic
+  description: whether it is mutagenic (1) or not mutagenic (0)
+  units: ld50
+  type: categorical
+  names:
+  - mutagenic
+  - Mutagenicity
+  - Ames Mutagenicity
```

Comment:
can we write out this acronym?

Reply:
Prof. Ames? ;-)

"the most widely used assay for testing the mutagenicity of compounds is the Ames experiment which was invented by a professor named Ames"
https://tdcommons.ai/single_pred_tasks/tox/#ames-mutagenicity